### PR TITLE
Add Slovak translation

### PR DESCRIPTION
**Pull request by daaAd1**
Add translation for slovak language. From changelog folder translates only 1.6 changelog and next.md.

### CODE REVIEW

File: assets/texts/sk/Changelog/1.2.1.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<ul>
```
Comment (rugk) [inline assets/texts/sk/Changelog/1.2.1.html]: If you have untranslated files, then please just remove them. (i.e. no English files in the `sk` dir) AMO automatically falls back to English. (and I need to upload it manually, anyway)

File: assets/texts/sk/Changelog/1.6.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+* **New:** The QR code now shows a warning if you try to enter too much text. ([#168](https://github.com/rugk/offline-qr-code/pull/168), thanks [@ongspxm](https://github.com/ongspxm))
```
Comment (rugk) [inline assets/texts/sk/Changelog/1.6.md]: also remove this (it's okay not to translate the md file, as you did correctly)

File: assets/texts/sk/amoDescription.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
+In contrast to many other add-ons, which use Google Web APIs for generating QR codes, this add-on works completely offline. <b>This QR code generator puts your privacy first! 🔐</b>
```
Comment (rugk) [inline assets/texts/sk/amoDescription.html]: This first sentence is not translated.

**Comment by rugk** [comment]
Hi @daaAd1,
thanks for your first contribution to this project! :tada: :+1: 🏅
I hope you'll like this project and enjoy hacking on it… :smiley:

I've just commented on some minor issues with the files. But generally, it looks good and adds another language. Yay! :+1: 

You can already subscribe to https://github.com/rugk/offline-qr-code/issues/138 to get notified for new updates on translations etc. :smiley:

**Comment by daaAd1** [comment]
Should be fixed now, I also subscribed to #138 to the issue.

**Comment by daaAd1** [comment]
Should be fixed now, although i have now changes from main repo in there, not sure if its a problem.

File: src/options/options.html [hunk 4]
```diff
@@ -162,6 +155,16 @@ <h2 data-i18n="__MSG_subtitleMisc__">Miscellaneous settings</h2>
 						</div>
 					</li>
 
+					<li class="message-container">
```
Comment (rugk) [inline src/options/options.html]: Hmm, this change likely should not belong in this PR. :thinking:

**Comment by rugk** [comment]
8d32fa6e025be803aad3cacf4fca9e8320d871cd includes the changes from master, this is strange. Maybe you can revert this commit (and only extract the changes you did manually), so not all this changes are attributed to you… :thinking: 

Or well… [you just use git in the "simple" way](https://xkcd.com/1597/) and redo the PR… :wink:

**Comment by daaAd1** [comment]
Merged with your original repo, should be ok now. Let me know if you need anything else changed here.
